src/index.html

```diff
@@ -15,8 +15,96 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="MOYO"
+        />
+      </a>
+      <nav>
+        <ul class="menu-list">
+          <li class="list-item">
+            <a
+              href="/"
```

Comment:
The `href` attribute for the menu link is set to '/', which might be a placeholder. Ensure it is replaced with the correct URL or path for proper navigation functionality. This applies to all menu links.